spec/datadog/ci/test_spec.rb

```diff
@@ -157,6 +157,70 @@
           .to eq(test_type)
       end
     end
+
+    context 'with environment runtime information' do
+      context 'for the architecture platform' do
+        subject(:tag) do
+          set_tags!
+          span.get_tag(Datadog::CI::Ext::Test::TAG_OS_ARCHITECTURE)
+        end
+
+        it { is_expected.to eq('x86_64') }
```

Comment:
Hmm this is a bit hardcodey. I suggest something testing `['i686', 'x86_64', 'aarch64', 'arm', 'arm64'].include?(...)`.

It doesn't cover everything, but would probably break less often :)

References:
* `arm64` seems to be the correct for the new arm macs, discussed in https://bugs.ruby-lang.org/issues/17021
* `arm` / `aarch64` is what my Android phones report (yes I have ruby on them, judge me!)
* `i686` is what I got from running Ruby on the `i386/ubuntu:latest` docker image.